src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-zonegroup-form/rgw-multisite-zonegroup-form.component.html

```diff
@@ -1,208 +1,255 @@
-<cd-modal [modalRef]="activeModal">
-  <ng-container i18n="form title"
-                class="modal-title">{{ action | titlecase }} Zone Group</ng-container>
-
-  <ng-container class="modal-content">
-    <form name="multisiteZonegroupForm"
-          #formDir="ngForm"
-          [formGroup]="multisiteZonegroupForm"
-          novalidate>
-    <div class="modal-body">
-      <div class="form-group row">
-        <label class="cd-col-form-label"
-               for="selectedRealm"
-               i18n>Select Realm</label>
-        <div class="cd-col-form-input">
-          <select class="form-select"
-                  id="selectedRealm"
-                  formControlName="selectedRealm"
-                  name="selectedRealm">
-          <option ngValue=""
+<cds-modal size="xl"
+           [open]="open"
+           [hasScrollingContent]="true"
+           (overlaySelected)="closeModal()">
+  <cds-modal-header (closeSelect)="closeModal()">
+    <h3 cdsModalHeaderHeading
+        i18n>{{ action | titlecase }} Zone Group</h3>
```

Comment:
Title does not match submit button text

<img width="972" height="808" alt="Image" src="https://github.com/user-attachments/assets/9154cfd6-5b0e-49c8-bdd5-c998436878b1" />